protos/vm.proto

```diff
@@ -113,3 +113,18 @@ message VMExecuteRequest {
 service VMService {
     rpc ExecuteContracts (VMExecuteRequest) returns (VMExecuteResponses) {}
 }
+
+message MvIrSourceFile {
+    bytes text = 1;
+    bytes address = 2;
+    ContractType type = 3;
+}
+
+message CompilationResult {
+    bytes bytecode = 1;
+    repeated bytes errors = 2;
```

Comment:
Missed documentation. Please describe what in this bytes is.
E.g.: "UTF-encoded string without trailing zero".